package.json

```diff
@@ -41,14 +41,15 @@
   },
   "build": {
     "appId": "com.symphony.electron_desktop",
-    "uninstallDisplayName": "${productName}",
     "artifactName": "${productName}-${version}-${os}.${ext}",
     "asar": true,
     "asarUnpack": "**/*.node",
     "nsis": {
       "perMachine": true,
```

Comment:
[Question] does it mean we would need admin rights to perform an auto update?

Reply:
Admin permission is required only if the user is installing in the program files directory

`perMachine` and `oneClick` will be **false** and the user will be shown an option to choose the install directory